### DISCUSSION TITLE
Docker image for v3.0 beta

### DISCUSSION THREAD

**Discussion by faab007nl** [requarks/wiki#5361] [Help / Questions] [answered]
Is there a docker image available for the v3 version?

**Comment by 573077632** [comment, marked as answer]
Not yet ready to release beta.

**Reply by faab007nl** [reply]
Oke.
The ETA for v3 is H2 2022 right?

**Reply by Greazi-Times** [reply]
Yea, I also would like to know if this is possible

**Reply by YunusEmreAlps** [reply]
When I was running the vega branch, I'm facing a database connection error.
How can I run this vega branch?
